gravitee-apim-e2e/ui-test/support/commands.ts

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
do we need to keep this ?

Reply:
Ok, I'll remove these comments and put references to the actual command definitions there (had them in e2e.ts file before) as this the place where they supposed to be.